manifests/devconsole/0.1.0/devconsole_v1alpha1_component_crd.yaml

```diff
@@ -31,6 +31,13 @@ spec:
               - nodejs
             codebase:
               type: string
+              description: The git url of your component source code.
+            port:
+              type: string
```

Comment:
How about type `integer` and having some min and max limit - `65635`?

Reply:
good idea, implemented 425b8f4 with a range [1024-65535]